src/main/java/com/minecolonies/coremod/entity/ai/citizen/crusher/EntityAIWorkCrusher.java

```diff
@@ -86,9 +85,6 @@ protected IAIState crush()
         {
             return getState();
         }
-        WorkerUtil.faceBlock(getOwnBuilding().getPosition(), worker);
```

Comment:
why did you remove faceBlock?

Reply:
causes the face to juggle left/right all the time, it only works when set each tick

Reply:
but now, does he still face the block?

Reply:
probably not looking down to it anymore, but he's got the right direction from the pathing

Reply:
Quite often when I observed him before adding this he was facing the opposite direction =P